seeds/dex/trades/dex_trades_seed.csv

```diff
@@ -295,4 +295,5 @@ bnb,pancakeswap,stableswap,2022-09-30 0:00,0xc8e562b51d15bc7dcc461f0d8f674c368cb
 bnb,pancakeswap,stableswap,2023-02-23 0:00,0x926a9463e8b1c94b267f8996c0aa3d25f0650b16d84cdf22c5c8cbd175b35a92,95,0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d,29.52590913,0xe9e7cea3dedca5984780bafc599bd69add087d56,29.53001786
 bnb,pancakeswap,stableswap,2023-02-23 0:00,0x2748ee8746f8ef3188f256312489cf877f819725c7b8526eabd2588437bced33,318,0xe9e7cea3dedca5984780bafc599bd69add087d56,17.00035523,0x55d398326f99059ff775485246999027b3197955,17
 bnb,pancakeswap,stableswap,2023-02-23 0:00,0xac4971ca3d750e40957720b9ed7d312de71868da236c5775f6cee8dc6a14bf30,226,0x55d398326f99059ff775485246999027b3197955,1000.607238,0xe9e7cea3dedca5984780bafc599bd69add087d56,1000.83
-bnb,pancakeswap,stableswap,2023-02-23 0:00,0xe71f7a8812df06a0b36b88a2746d1d48ad39bb5bcb61c23572aeea04df96cc50,181,0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d,2499.859114,0x55d398326f99059ff775485246999027b3197955,2500
\ No newline at end of file
+bnb,pancakeswap,stableswap,2023-02-23 0:00,0xe71f7a8812df06a0b36b88a2746d1d48ad39bb5bcb61c23572aeea04df96cc50,181,0x8ac76a51cc950d9822d68b83fe1ad97b32cd580d,2499.859114,0x55d398326f99059ff775485246999027b3197955,2500
```

Comment:
Please append the seed you deleted in another file to here, as we can still apply the `check_dex_seed` test.